anago

```diff
@@ -969,7 +969,7 @@ announce () {
     # so values > 0 are actually truthy
     ((FLAGS_nomock)) && nomock_flag="--nomock"
 
-    logecho "$ release-notify ${nomock_flag} ${RELEASE_VERSION_PRIME}"
+    logecho "$ krel announce ${nomock_flag} --tag ${RELEASE_VERSION_PRIME}"
     logecho
   else
     release::send_announcement || return 1
```

Comment:
Remove and outdent the above block since it's no longer a conditional.

Reply:
That is to say, we always want to get this output and we always run in GCB.

Reply:
I added the change to this PR. Do you think we can remove the `FLAGS_gcb` completely from anago? Would make things easier to read.

Reply:
Yep, that's what I was implying with outdent mention in this comment: https://github.com/kubernetes/release/pull/1323#discussion_r432453028 :)

Reply:
Got it changed the code part to remove the whole `if`